deps/v8/third_party/ittapi/CMakeLists.txt

```diff
@@ -0,0 +1,135 @@
+#/* <copyright>
+#  This file is provided under a dual BSD/GPLv2 license.  When using or
+#  redistributing this file, you may do so under either license.
+#
+#  GPL LICENSE SUMMARY
+#
+#  Copyright (c) 2005-2017 Intel Corporation. All rights reserved.
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of version 2 of the GNU General Public License as
+#  published by the Free Software Foundation.
+#
+#  This program is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin St - Fifth Floor, Boston, MA 02110-1301 USA.
+#  The full GNU General Public License is included in this distribution
+#  in the file called LICENSE.GPL.
+#
+#  Contact Information:
+#  http://software.intel.com/en-us/articles/intel-vtune-amplifier-xe/
+#
+#  BSD LICENSE
```

Comment:
I think we need an addition to  https://github.com/nodejs/node/blob/master/LICENSE which says that ittapi is licenced under BSD.

Reply:
Do we need to add it to the main LICENSE even if we don't distribute any Node.js release with this dependency?